pkgs/tools/security/ssh-audit/default.nix

```diff
@@ -0,0 +1,55 @@
+{ fetchFromGitHub, python3Packages, stdenv }:
+
+python3Packages.buildPythonPackage rec {
```

Comment:
From the looks this is simply an application and not a library, so you should use `buildPythonApplication` instead.

Reply:
exports also a library: `import ssh-audit`

Reply:
Didn't see that, thanks!